game/scripts/npc/abilities/centaur_hoof_stomp.txt

```diff
@@ -35,7 +35,7 @@
       "02"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "stun_duration"                                   "1.75 2.0 2.25 2.5 2.65 2.75" //OAA
+        "stun_duration"                                   "2.0 2.25 2.5 2.75 2.75 2.75"
```

Comment:
Talent value is back to normal. It was +1s, now its +1.5s. I think that change was a good buff already. 4.25s AoE stun in mid game is not cool. In late game you have enough status resistance so its fine. Scaling it from 2.0 at lvl 1 to 2.75 at lvl 6 would be ideal: 2/2.15/2.3/2.45/2.6/2.75. It buffs its early game, not cancer in mid game, and normal at lategame. But I also think current scaling (1.75/2./2.25/2.5/2.65/2.75) is fine as well. :man_shrugging:

Reply:
ok